File: move_assets.js
const fs = require('fs');
const path = require('path');

const src = 'c:\\Users\\ASUS\\Desktop\\Personal\\projects\\Personal Portfolio\\Portfolio\\src\\assets\\Mockups';
const dst = 'c:\\Users\\ASUS\\Desktop\\Personal\\projects\\Personal Portfolio\\Portfolio\\public\\Mockups';

function copyRecursiveSync(src, dest) {
  const exists = fs.existsSync(src);
  const stats = exists && fs.statSync(src);
  const isDirectory = exists && stats.isDirectory();
  if (isDirectory) {
    if (!fs.existsSync(dest)) {
      fs.mkdirSync(dest, { recursive: true });
    }
    fs.readdirSync(src).forEach(function(childItemName) {
      copyRecursiveSync(path.join(src, childItemName),
                        path.join(dest, childItemName));
    });
  } else {
    fs.copyFileSync(src, dest);
    console.log(`Copied ${src} to ${dest}`);
  }
}

if (fs.existsSync(src)) {
  copyRecursiveSync(src, dst);
  console.log('Finished moving files');
} else {
  console.log('Source not found');
}

File: move_assets.py
import shutil
import os

src = r"c:\Users\ASUS\Desktop\Personal\projects\Personal Portfolio\Portfolio\src\assets\Mockups"
dst = r"c:\Users\ASUS\Desktop\Personal\projects\Personal Portfolio\Portfolio\public\Mockups"

if os.path.exists(src):
    for root, dirs, files in os.walk(src):
        relative_path = os.path.relpath(root, src)
        dest_path = os.path.join(dst, relative_path)
        
        if not os.path.exists(dest_path):
            os.makedirs(dest_path)
            
        for file in files:
            shutil.copy2(os.path.join(root, file), os.path.join(dest_path, file))
            print(f"Copied {os.path.join(relative_path, file)}")
    
    # Optionally remove src after confirming copy
    # shutil.rmtree(src)
    print("Copy completed.")
else:
    print("Source directory does not exist.")
